Modules/DataModel/Sources/PocketCastsDataModel/Public/Bookmarks/BookmarkDataManager.swift

```diff
@@ -101,6 +101,28 @@ public struct BookmarkDataManager {
         selectBookmarks(where: [.deleted], values: [includeDeleted], sorted: sorted)
     }
 
+    /// Returns the total number of undeleted bookmarks for the episode
```

Comment:
Wdyt about changing to something like:

```suggestion
    /// Returns the total number of bookmarks for the episode
    /// This won't return the number of bookmarks still on the database for this episode that was deleted
```

Reply:
I was thinking about it and to be more inline with the other methods I added a flag to optionally include the deleted items.